src/screens/SignUp/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+
+import { email, validatePassCoincidence } from 'utils/inputValidations';
+
+import wLogo from '../../assets/wLogo.png';
+
+import 'scss/layout.scss';
+import 'scss/components.scss';
+import 'scss/margins.scss';
+import styles from './styles.module.scss';
+
+interface Inputs {
+  name: string;
+  lastName: string;
+  email: string;
+  password: string;
+  confirmPassword: string;
+}
+
+function SignUp() {
+  const { register, errors, getValues, setError } = useForm<Inputs>();
+  //   const onSubmit = (data: Inputs) => console.log(data);
+  const onBlur = (): void => {
+    const response = email('correo no válido')(getValues('email'));
+    if (response) {
+      setError('email', {
+        message: response
+      });
+    }
+  };
+
+  return (
+    <div className="row full-height center">
+      <form
+        className={`column full-height center space-around ${styles.formContainer} ${styles.formBackground}`}
+      >
+        <img className={styles.formImage} src={wLogo} alt="wolox-logo" />
+        <div className="column full-width">
+          <label className={`m-bottom-1 ${styles.formLabel}`}>Nombre</label>
```

Comment:
add for attribute, also you can try something like list rendering 
![image](https://user-images.githubusercontent.com/26824183/107815022-c5468600-6d40-11eb-8c4a-0d5ff089c344.png)